Entitlements.plist

```diff
@@ -4,9 +4,9 @@
 <dict>
 	<key>com.apple.security.device.camera</key>
 	<true/>
-    <!-- <key>com.apple.developer.associated-domains</key>
+    <key>com.apple.developer.associated-domains</key>
```

Comment:
@igor-sirotin is the one that commented this because it was causing issues with the Mac builds. Are  those problems fixed now?

Reply:
Should be fixed together with added `provisionprofile` below.
@borismelnik should know better 🙂

Reply:
@jrainville @igor-sirotin I just trying to sign it correctly... For now its still not correct :(